mlflow/example/license.sig

```diff
@@ -0,0 +1 @@
+ENTER VALID DRIVERLESS AI LICENSE
```

Comment:
nit: do we need this file at all? I would expect based on the scoring code below + readme that this should be understood, but I am fine with keeping it.